### ISSUE TITLE
Please add "ExQuilla for Microsoft Exchange" and "TbSync (Provider for Exchange ActiveSync)" to ATN Featured Extensions

### ISSUE THREAD

**Issue by ovari** [thundernest/addons-server#13]
Could you please add "[ExQuilla for Microsoft Exchange](https://addons.thunderbird.net/addon/exquilla-exchange-web-services/)" and "[TbSync (Provider for Exchange ActiveSync)](https://addons.thunderbird.net/addon/tbsync/)" to ATN "[Featured Extensions](https://addons.thunderbird.net/extensions/?sort=featured)".

There are reviews like "[Works perfectly. Should be included by default in Thunderbird together with TBsync.](https://addons.thunderbird.net/addon/exquilla-exchange-web-services/reviews/1154436/)"

Both add-ons also work with Thunderbird 60.

Thank you

**Comment by kewisch** [comment]
We do not have a feature board for ATN yet, and instead of just adding extensions I think we should have @ryanleesipes organize one, with a similar process as for AMO

**Comment by ovari** [comment]
It would be appreciated if you would advise:
1. Can approval for the inclusion of these 2 add-ons in [Featured Extensions](https://addons.thunderbird.net/extensions/?sort=featured) be approved before a feature board is operational? Say at the next [Thunderbird/Status Meeting](https://wiki.mozilla.org/Thunderbird/StatusMeetings)?
2. The ETA of an operational feature board for [ATN](https://addons.thunderbird.net/extensions/?sort=featured)?

TL;DR

Currently there are 5 [Featured Extensions](https://addons.thunderbird.net/extensions/?sort=featured) on [ATN](https://addons.thunderbird.net/extensions/?sort=featured): [CardBook](https://addons.thunderbird.net/addon/cardbook/), [CompactHeader](https://addons.thunderbird.net/addon/compactheader/), [Enigmail](https://addons.thunderbird.net/addon/enigmail/), [Mail Merge](https://addons.thunderbird.net/addon/mail-merge/), and [Mail Redirect](https://addons.thunderbird.net/addon/mailredirect/).

It seems strange that the [Lightning](https://addons.thunderbird.net/addon/lightning/) add-on is not a featured add-on, yet is bundled in the default Thunderbird installation. Does that mean the Lightning add-on can not be marked as a [Featured Extension](https://addons.thunderbird.net/extensions/?sort=featured) until a feature board is organized? Would think not.

[AMO Featured Extensions Policies](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/AMO/Policy/Featured#Joining_the_Featured_Add-ons_Advisory_Board) states that:
“1. The extension must have a complete and informative listing on AMO [ATN]. This means:
** a 64-pixel custom icon”
Does that mean that [Mail Merge](https://addons.thunderbird.net/addon/mail-merge/) should be removed as a [Featured Extension](https://addons.thunderbird.net/extensions/?sort=featured)? Would think not.

[AMO Featured Extensions Policies](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/AMO/Policy/Featured#Joining_the_Featured_Add-ons_Advisory_Board) states that:
“1. The extension must have a complete and informative listing on AMO [ATN]. This means:
** updated screenshots of the extension's functionality”
Does that mean that [CardBook](https://addons.thunderbird.net/addon/cardbook/) should be removed as a [Featured Extension](https://addons.thunderbird.net/extensions/?sort=featured)? Would think not.

[AMO Featured Extensions Policies](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/AMO/Policy/Featured#Joining_the_Featured_Add-ons_Advisory_Board) states that:
“3. The extension must have a minimum of 500 users.”
Unfortunately add-on user stats are no longer showing with no ETA for a fix, see
[Parse Cloudfront logs for add-on user stats](https://github.com/thundernest/addons-server/issues/4). Does that mean new add-ons have no chance of being shown as a [Featured Extensions](https://addons.thunderbird.net/extensions/?sort=featured)? Would think not

[AMO Featured Extensions Policies](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/AMO/Policy/Featured#Joining_the_Featured_Add-ons_Advisory_Board) states that:
“6. **Most importantly**, the extension must have wide consumer appeal to Firefox’s [Thunderbird’s] users and be outstanding in nearly every way: user experience, performance, security, and usefulness or entertainment value.”

Enabling these add-ons to be found in the [Featured Extensions](https://addons.thunderbird.net/extensions/?sort=featured) before the release of Thunderbird 60:
1. Can provide great marketing value as there can be interest in Thunderbird 60 as it is an ESR and hence may have more coverage by journals and uptake by users
2. Can help individuals and organizations looking to migrate away from increased costs and vendor lock-in, eg. “[Microsoft raises prices on Office and server products by 10 percent](https://www.techspot.com/community/topics/microsoft-raises-prices-on-office-and-server-products-by-10-percent.248100/)”

Thank you
